contracts/interfaces/RevokeAndPublish.sol

```diff
@@ -0,0 +1,32 @@
+pragma solidity 0.4.19;
+import '../EthereumClaimsRegistry.sol';
+
```

Comment:
Why is this contract in an “interfaces” folder? Not sure I would call it an “interface”?

Reply:
Yeah, not sure about the right terminology here. Do you have a better name in mind that describes something that provides a way of interacting with the ECR?

Reply:
Changed it to "extensions" now. Not sure that it's ideal though.

Reply:
Not sure what good terminology is, maybe "applications"? It's kind of a contract that uses the registry as a dependency.

Reply:
yeah, applications seems better at least.